### PR TITLE
Added Benchmark file with profiling

### PR DESCRIPTION
**Pull request by nilabha**
Benchmark file for use with New Issue created 
https://github.com/spMohanty/RogiRL/issues/11

### CODE REVIEW

**Comment by spMohanty** [comment]
@nilabha : Can you expose this via Click cli ? 
We could something like : rogi-rl-profile-perf which can run this script and gather all the data needed. 

At the later point of time we can also look at more stuff like CPU util, men usage, etc etc

**Comment by nilabha** [comment]
> @nilabha : Can you expose this via Click cli ?
> We could something like : rogi-rl-profile-perf which can run this script and gather all the data needed.
> 
> At the later point of time we can also look at more stuff like CPU util, men usage, etc etc

@spMohanty 
I have made the changes. We can get the performance metrics by running rogi-rl-profile_perf. For profiling of render use rogi-rl-profile_perf -p

Additionally I have registered the env with gym so we can use the gym.make(..) and fixed some close renderer issues.

File: rogi_rl/cli.py [hunk 1]
```diff
@@ -22,5 +22,21 @@ def demo(width, height):
     server.launch()
 
 
+@click.command()
+@click.option('-p/-np', '--profile-render/--no-profile-render',
```
Comment (spMohanty) [inline rogi_rl/cli.py:26]: Should we just use something like : `--render-on` ?
It will be more intuitive to understand.

Comment (nilabha) [reply rogi_rl/cli.py]: I have made the changes

**Comment by spMohanty** [comment]
This looks great now ! Merging into master !

**Pull request merged**